tsec-http4s/src/main/scala/tsec/authentication/CookieAuthenticator.scala

```diff
@@ -17,7 +17,7 @@ import cats.syntax.all._
 import scala.concurrent.duration.FiniteDuration
 
 abstract class CookieAuthenticator[F[_], Alg: MacTag: ByteEV, I, V]
-    extends AuthenticatorEV[F, Alg, I, V, AuthenticatedCookie[?, I]]
+    extends Authenticator[F, Alg, I, V, AuthenticatedCookie[?, I]]
```

Comment:
inside `renew` and `refresh` in the case `Some(idletime)`, value idletime is ignored. Is this a correct behavior ?